fetch.bs

```diff
@@ -2230,6 +2230,8 @@ or "<code>object</code>".
 return true:
 
 <ol>
+ <li> Assert: <var>request</var>'s <a for=request>origin</a> is not "<code>client</code>".
```

Comment:
```suggestion
 <li><p><a for=/>Assert</a>: <var>request</var>'s <a for=request>origin</a> is not
 "<code>client</code>".
```

Reply:
In particular, no space after `<li>` but it also needs a nested `<p>` like the other `<li>`s, we want to link Assert as people can use the additional guidance that provides, and then we need to wrap at a 100 columns unfortunately due to the additional text.

If you apply this throughout I'd be happy to merge this. Thanks for turning it around so quickly!

Reply:
Done - I've also added links for `Assert:` in all places that did not previously have them